README.md

```diff
@@ -5,5 +5,12 @@ Repo for Administrator
 1. If you do not have maven installed, you need to install it
 2. Import this project as a maven project
 3. Set up MySQL schema using the schema in src/test/resources/schema.sql
-4. Change root and password in src/main/resources/application.properties
-        to access your MySQL server with the library schema
+4. Create a file called `database-config.properties` in src/main/resources. Look at the code down below for an example. <br/>
+```
```

Comment:
It would be nice to get guaranteed-correct syntax highlighting by specifying the language here, but [the documentation](https://help.github.com/en/articles/creating-and-highlighting-code-blocks#syntax-highlighting) and [the languages YAML file](https://github.com/github/linguist/blob/master/lib/linguist/languages.yml) it links to aren't enough for me, at least, to figure out what we should put here to tell it "this is a Java properties file".

Reply:
I have added a comment right before the code stating what language it is.

It does not show it here. But if you look at the files changed. It should show up there

Reply:
The point of specifying the language isn't (directly) for *people* reading the code, but so that the GitHub rendering of it will apply syntax coloring.

For example, the difference between these two snippets:

```
int i = 0;
```

```java
int i = 0;
```

GitHub tries to detect languages, but if we could specify it, we would be assured that it wouldn't detect the wrong language.

Reply:
(And "which is in Java" isn't really true, since the Java properties format isn't Java *code* and [is also used elsewhere](https://en.wikipedia.org/wiki/.properties#Non-Java_Uses_and_Exceptions), and at most applies to the *file*, not the connection as the way you wrote the sentence implies.)

Reply:
Okay, got it. This should have better highlighting than before

Reply:
Okay, so should I remove it. You are right it is a config file. So do you want me to remove it? And also the Java after the ``` to signify that for syntax highlighting?

Reply:
I have made the changes you recommended.